compiler-rt/test/asan/TestCases/Windows/aligned_alloc.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cl_asan %Od %s %Fe%t
```

Comment:
This basically does the same as compiler-rt/test/asan/TestCases/Windows/aligned_mallocs.cpp
But like the test you're adding, that only checks that we can call the functions and get aligned memory back. It doesn't check that we're intercepting the system's implementations.

We want a test that fails if we fail to intercept the functions, and passes otherwise. How about something like using _aligned_malloc for a 128-byte chunk of memory, trying to write to one byte *before* the address and checking that we get a proper asan error?